frontend/static/js/helper.js

```diff
@@ -130,11 +130,11 @@ class Helper {
 
             let innerHTML = '';
             innerHTML += '<div data-hover="" data-delay="0" class="accordion-item">';
-            innerHTML += '<div class="accordion-toggle" onclick="helper.toggleSection(this)">';
+            innerHTML += '<div class="accordion-toggle" disabled="true" onclick="helper.toggleSection(this)">';
             innerHTML += '<div class="accordion-icon"></div>';
             innerHTML += ('<div class="features-label">' + feature['name'] + '</div>');
             innerHTML += '</div>';
-            innerHTML += '<nav class="dropdown-list">';
+            innerHTML += '<nav class="dropdown-list" style="height: 0px;" data-collapsed="true">';
```

Comment:
Inline css bitte noch entfernen @twonkos

Reply:
Das gehört zur Funktion der "flüssig aufklappenden" accordions -> Modifikation des heights über JS

Background (Warum mit JS, und nicht mit CSS/HTML only) ->
Warum das mit JS gemacht werden muss und nicht über HTML/CSS only, hier nachzulesen -> https://css-tricks.com/using-css-transitions-auto-dimensions/

Reply:
TL;DR:

Wenn man das in CSS/HTML only macht, muss man die height hardcoden -> bad practice
-> und nicht mehr responsive

Reply:
Ich glaub ich hab jetzt erst deinen Punkt verstanden, pardon! Wird entfernt